graphkb/genes.py

```diff
@@ -161,3 +162,200 @@ def get_genes_from_variant_types(
         ),
     )
     return result
+
+
+def get_preferred_gene_name(conn: GraphKBConnection, gene_name: str, source: str = '#39:5') -> str:
```

Comment:
It'd be helpful to move `'#39:5'` to constants and add a comment about what it is. Something like `PREFERRED_GENE_SOURCE = "#39.5"  # HGNC`